WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -10,7 +10,12 @@ final class NewOrderHostingController: UIHostingController<NewOrder> {
     private var subscriptions: Set<AnyCancellable> = []
 
     init(viewModel: NewOrderViewModel) {
-        super.init(rootView: NewOrder(viewModel: viewModel))
+        let view = NewOrder(viewModel: viewModel)
+        super.init(rootView: view)
+    }
+
+    func setDismissAction(_ dismissAction: @escaping () -> Void) {
+        rootView.dismissAction = dismissAction
     }
```

Comment:
We could simplify this by setting the dismiss action in the `init` method, rather than having a separate function for it, similar to how we do for the `SimplePaymentsAmount` view:

https://github.com/woocommerce/woocommerce-ios/blob/c04248d75b06b25c318ba4e60443521ecb1e259b/WooCommerce/Classes/ViewRelated/Orders/Simple%20Payments/Amount/SimplePaymentsAmount.swift#L31-L34

Reply:
Nice suggestion! Updated in d6dabf46b46ddbc1b6d61b06b37700416882b86a 👍